introduction.md

```diff
@@ -1 +1,6 @@
 hidashjldjkhalskjhglakjhr ajskflhkj asdhjfklk ajhdklfjh lasdjkfh i don't like this part of training. I liked learning about regex in Electives
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?